### ISSUE TITLE
Thread functionality for messages

### ISSUE THREAD

**Issue by realrill** [vector-im/element-web#18288]
#### Is your suggestion related to a problem? Please describe.
No

#### Describe the solution you'd like.
Not sure if it's trademarked or not, but "Slack" like `thread` would be awesome. Reply is good for simple y/n responses, but for long running discussions it becomes super hard to follow which reply is for what.

#### Describe alternatives you've considered.
N/A

#### Additional context
[Sample picture that I found on ddg](https://www.turn-on.de/media/cache/article_image_slider/media/cms/2017/01/Slack-Threads-02.jpg?810887)

Unsure if it's Element or Synapse related really. Also, if this feature request is a duplicate, feel free to close.

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/2349
